extension/js/common/browser/browser-msg.ts

```diff
@@ -446,6 +451,40 @@ export class BrowserMsg {
     });
   }
 
+  public static thunderbirdSecureComposeHandler() {
+    const handleClickEvent = async (tabId: number, acctEmail: string, thunderbirdMsgId: number, composeMethod?: browser.compose._ComposeDetailsType) => {
+      const accountEmails = (await GlobalStore.get(['account_emails'])).account_emails;
+      const useFullScreenSecureCompose = (await browser.windows.getCurrent()).type === 'messageCompose';
```

Comment:
Hi @sosnovsky I think I have pretty much everything in this PR. Just one thing for me to do, I need a better way of detecting the account that has FlowCrypt successfully setup more than the account that is already connected with FlowCrypt. I have checked that the "await GlobalStore.get(['account_emails'])" just lists all the account connected to FlowCrypt - that includes previously setup account but then decided to abandon setup such as performing account reset doesnt seem to deduce that array.

Other than that, it can now be reviewed in terms of usage. Please observe, quote rendering, visuals, ability to identify reply or forward, rendering of email based on threadId.

Thank you for help.